sink-connector/python/db_load/clickhouse_loader.py

```diff
@@ -194,6 +194,8 @@ def convert_to_clickhouse_table(user_name, table_name, source):
     # ClickHouse does not support constraints, indices, primary and unique keys
     src = re.sub(r'.*\bCONSTRAINT\b.*', '', src)
     src = re.sub(r'.*\bPRIMARY KEY\b.*\(.*', '', src)
+    # COMMENT is not supported
+    src = re.sub(r'\s*\bCOMMENT\b\s*\'(.*?)\'', '', src)
     # primary key on the column itself
```

Comment:
feel free to add this to the code, it is a good addition to the regexp parser (there is a more elaborate ANTLR parser that kicks in first now), so it would only be called if you explicitly enable the regexp parser of fi there is an exception in the ANTL parser.